satnogsclient/scheduler/tasks.py

```diff
@@ -30,9 +30,19 @@ def spawn_observation(*args, **kwargs):
 
 def get_jobs():
     """Query SatNOGS Network API to GET jobs."""
+
+    print 'Try getting Jobs from Network ..'
+
     url = urljoin(settings.NETWORK_API_URL, 'jobs')
     params = {'ground_station': settings.GROUND_STATION_ID}
-    response = requests.get(url, params=params)
+    try:
+        response = requests.get(url, params=params, verify=settings.CA_CERT)
+    except:
+        print 'HTTPs certificate verification failed..(ignore for now)'
+        response = requests.get(url, params=params, verify=False)
```

Comment:
This discussion is mostly in https://github.com/satnogs/satnogs-client/issues/37 .
This is a dirty workaround to get it working. In theory you should be able to provide a CA_CERT file path yourself, however in no scenario I could get this working at all.